### ISSUE TITLE
[FIX] Aggregate API for refunds to use JWT authentication only, removing API Key auth

### ISSUE THREAD

**Issue by apoorvdixit88** [juspay/hyperswitch#5989]
### Feature Description

Refactor `/refunds/aggregate` API to use JWT authentication only, removing API Key auth

The /refunds/aggregate API is meant for internal use only and should not be accessible via API Key authentication. It should be secured using JWT authentication exclusively, as it is not intended to be used by merchants.

Current Behavior:
- The API currently supports both API Key and JWT authentication.
Expected Behavior:
- Refactor the API to allow JWT authentication only, enforcing that only internal users can access this endpoint.
- Remove the API Key authentication block as merchants should not have access to this API.

### Possible Implementation

The route function [get_refunds_aggregates](https://github.com/juspay/hyperswitch/blob/37925626e6446900f1d16e0e5f184ee472d4be3e/crates/router/src/routes/refunds.rs#L426) should be modified to remove API Key authentication and ensure that only JWT auth is applied.

The route function
```
pub async fn get_refunds_aggregates(
    state: web::Data<AppState>,
    req: HttpRequest,
    query_params: web::Query<common_utils::types::TimeRange>,
) -> HttpResponse
```

Hint: Additional context can be taken from paymets aggregate function.

### Have you spent some time checking if this feature request has been raised before?

- [X] I checked and didn't find a similar issue

### Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/juspay/hyperswitch/blob/main/docs/CONTRIBUTING.md)

### Are you willing to submit a PR?

None

**Comment by sangram-mkj-01** [comment]
Hi, can you assign me to this issue?

**Comment by gorakhnathy7** [comment]
Hey @sangram-mkj-01 you had requested for another issue earlier, which has been assigned to you.
Kindly let me know, once that is done, i can assign another one to you!

**Comment by jayantp2003** [comment]
I would like to work on this issue.

**Comment by gorakhnathy7** [comment]
Sure @jayantp2003 Assigning the issue to you!

**Comment by apoorvdixit88** [comment]
Hey @jayantp2003 are you working on this issue?

**Comment by gorakhnathy7** [comment]
Hey @apoorvdixit88 Kindly free this up, if anyone is interested.

**Comment by aniketh-varma** [comment]
Hi @gorakhnathy7, I would like to take up this issue, if it's up for grabs

**Comment by gorakhnathy7** [comment]
Sure, @aniketh-varma Assigning this to you, thanks!
